packages/story-editor/src/components/header/title.js

```diff
@@ -43,6 +43,8 @@ const Input = styled.input`
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+  ${({ theme }) => themeHelpers.focusableOutlineCSS(theme.colors.border.focus)};
```

Comment:
🤔 I think `theme.colors.border.focus` is the default, so does the following work?
```suggestion
  ${themeHelpers.focusableOutlineCSS};
```